DataFormats/GEMDigi/BuildFile.xml

```diff
@@ -1,5 +1,7 @@
 <use   name="DataFormats/Common"/>
+<use   name="DataFormats/GeometryVector"/>
 <use   name="DataFormats/MuonDetId"/>
+<use   name="rootrflx"/>
```

Comment:
this was previously cleaned up in #8090 
https://github.com/cms-sw/cmssw/pull/8090/files#diff-f926bac44e8a8673b0eed81dd1550b8fL4
I think there should be no changes in this file (this is the only change in DataFormats/GEMDigi and we didn't need it to compile so far)